src/client/app/admin/views/instance.vue

```diff
@@ -54,7 +54,7 @@
 			</ui-horizon-group>
 			<ui-horizon-group inputs>
 				<ui-input v-model="smtpUser" :disabled="!enableEmail">{{ $t('smtp-user') }}</ui-input>
-				<ui-input v-model="smtpPass" :disabled="!enableEmail">{{ $t('smtp-pass') }}</ui-input>
+				<ui-input v-model="smtpPass" :type="'password'" :disabled="!enableEmail">{{ $t('smtp-pass') }}</ui-input>
```

Comment:
I think `type="password"` is better expression than `:type="'password'"`.

Reply:
`:type="'password'"` not working